lightning/src/chain/channelmonitor.rs

```diff
@@ -253,18 +253,13 @@ pub const ANTI_REORG_DELAY: u32 = 6;
 ///    fail this HTLC,
 /// 2) if we receive an HTLC within this many blocks of its expiry (plus one to avoid a race
 ///    condition with the above), we will fail this HTLC without telling the user we received it,
-/// 3) if we are waiting on a connection or a channel state update to send an HTLC to a peer, and
-///    that HTLC expires within this many blocks, we will simply fail the HTLC instead.
```

Comment:
Are these comment removals related to the main change? The check that was changed doesn't seem related to awaiting on a peer connection/state update..

Reply:
Ah! Good catch - so these two things are really the same thing, but this comment was *actually* referring to a check in `channel` and not in `channelmanager`, even though they're the same check. I've updated the check in channel as well.

Reply:
Should we mirror this in `ChannelManager::do_chain_event` as well? It still uses `HTLC_FAIL_BACK_BUFFER` to check for timed out HTLCs

Reply:
No, that one is about our own incoming HTLCs, not forwarded ones - that is case `2`.